data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -1096,6 +1148,12 @@ impl TraceExporterBuilder {
         self
     }
 
+    /// Enables storing and checking the agent payload
+    pub fn enable_agent_rates_payload_version_enabled(&mut self) -> &mut Self {
```

Comment:
This feels very strange to `enable...enabled`. I think you should skip the last `enabled`.